### PR TITLE
MWPW-141104: Consume Milo SWC in the unit tests

### PR DESCRIPTION
**Pull request by yesil**
Consume Spectrum Web Components from Milo in CC unit tests

Resolves: [MWPW-141104](https://jira.corp.adobe.com/browse/MWPW-141104)

**Test URLs:**

### CODE REVIEW

File: creativecloud/scripts/utils.js [hunk 1]
```diff
@@ -22,7 +22,11 @@
 export const [setLibs, getLibs] = (() => {
   let libs;
   return [
-    (prodLibs) => {
+    (prodLibs, force = false) => {
```
Comment (npeltier) [inline creativecloud/scripts/utils.js:25]: i would still as discussed call this with a more explicit name on what exactly does it force :) 

also i wonder what is the point of having 
`const foo = getLibs('blah', true)` vs `const foo = 'blah'`

File: package.json [hunk 2]
```diff
@@ -39,5 +39,8 @@
     "stylelint": "14.6.0",
     "stylelint-config-prettier": "9.0.3",
     "stylelint-config-standard": "25.0.0"
+  },
+  "dependencies": {
+    "@adobecom/milo": "github:adobecom/milo#main"
```
Comment (npeltier) [inline package.json:44]: ❤️

**Review by npeltier — APPROVED** [review]

File: creativecloud/scripts/utils.js (hunk 1 above)
Comment (yesil) [reply creativecloud/scripts/utils.js:25]: there is no `const foo = getLibs('blah', true)` 
only `const foo = setLibs('blah', true)`

Comment (yesil) [reply creativecloud/scripts/utils.js:25]: force is pretty self explanatory no?

**Comment by npeltier** [comment]
merging as commercev3

**Pull request merged**